src/Bridges/Constraint/det.jl

```diff
@@ -446,15 +440,27 @@ function MOI.get(
     return [b.gmindex]
 end
 
-# References
 function MOI.delete(model::MOI.ModelLike, bridge::RootDetBridge)
     MOI.delete(model, bridge.gmindex)
     MOI.delete(model, bridge.sdindex)
     MOI.delete(model, bridge.Δ)
     return
 end
 
-# Attributes, Bridge acting as a constraint
+function MOI.supports(
```

Comment:
The setter for `ConstraintPrimalStart` is not yet implemented for the bridge so it does not support it yet

Reply:
Oops. I overlooked this. I just saw the `get` was implemented. I removed the `get` method for the start values then.